examples/chef/common/clusters/temperature-control/static-supported-temperature-levels.cpp

```diff
@@ -33,9 +33,12 @@ app::Clusters::TemperatureControl::AppSupportedTemperatureLevelsDelegate sAppSup
 CharSpan AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions[] = { "Low"_span, "Medium"_span, "High"_span };
 
 const AppSupportedTemperatureLevelsDelegate::EndpointPair AppSupportedTemperatureLevelsDelegate::supportedOptionsByEndpoints
-    [MATTER_DM_TEMPERATURE_CONTROL_CLUSTER_SERVER_ENDPOINT_COUNT] = { EndpointPair(
-        1 /* endpointId */, AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions,
-        MATTER_ARRAY_SIZE(AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions)) };
+    [MATTER_DM_TEMPERATURE_CONTROL_CLUSTER_SERVER_ENDPOINT_COUNT] = {
+        EndpointPair(1 /* endpointId */, AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions,
+                     MATTER_ARRAY_SIZE(AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions)),
+        EndpointPair(3 /* endpointId */, AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions,
+                     MATTER_ARRAY_SIZE(AppSupportedTemperatureLevelsDelegate::temperatureLevelOptions))
+    };
 
```

Comment:
Ovens MUST use TN (Temperature Number) feature, not TL (Temperature Level). See Matter 1.5 spec text on Temperatuer Controlled Cabinet and Oven device types

Reply:
Oven has Cooktop endpoint as Optional which has Cook surface endpoint with Temperature control cluster supported with Element TL: https://matter.teams.x20web.corp.google.com/connectedhomeip-spec/master/device_library.html#_element_requirements_30.

Should I not Add Cooktop Endpoint support for Oven? https://matter.teams.x20web.corp.google.com/connectedhomeip-spec/master/device_library.html#ref_OvenDeviceTypeRequirements